css/src/style.scss

```diff
@@ -0,0 +1,1703 @@
+//colors
+$red: #f00;								// warnings and errors
```

Comment:
You should use the core variables
https://github.com/nextcloud/server/blob/da5662c454cbdf9e81c6ac83cc0dfe72692fd883/core/css/variables.scss#L1-L8